definitions.yml

```
@@ -5,6 +5,8 @@ paths:
 definitions:
   Service:
     type: object
+    required:
+      - service_type
```

"type" is too generic, I would use something more specific, e.g. "scope" - also this is an attribute of the "Service" model, there's no need to prefix it with "service".

```suggestion
      - scope
```